test_fairlearn_smoke.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
So this adds the same file reformatted as pytest? What about the existing test_fairlearn.py?

Reply:
We might want to remove it eventually. I think that Miro would prefer to keep the existing test script as well for now